reverse_script.py

```diff
@@ -0,0 +1,64 @@
+import os, sys

```

Comment:
PEP8, one lib per line

Reply:
fixed in d608ea5dd37b093be0818a47c482cc653772dd49